### ISSUE TITLE
请问为什么这个就是target

### ISSUE THREAD

**Issue by Ashionnal** [calmisential/TensorFlow2.0_SSD#3]
Chinese:
你好请教一个问题
在label_anchors.py中get_result方法返回的offset_list的shape为(num_anchors, 4)， 也就是(8732, 4) 第二维度存放的是xmin, ymin, xmax, ymax， 这4个值是defaultbox的坐标与真值框的偏移值，那么为什么这个是作为在loss中的y_target呢，也就是你代码中reg_loss = train_reg_loss(box_target, box_preds)的box_target，我一直不理解这一块，其他我都懂了，麻烦大佬解答下

English:
Hi ask a question
The shape of the offset_list returned by the get_result method in label_anchors.py is (num_anchors, 4), which is (8732, 4). The second dimension stores xmin, ymin, xmax, and ymax. These four values are the coordinates of the defaultbox and true The offset value of the value box, so why is this the y_target in the loss, that is, the box_target in your code reg_loss = train_reg_loss (box_target, box_preds), I have not understood this one, I understand everything else, and it is a big trouble Guy answer
